🌐 [i18n-KO] Update README_ko.md

J4BEZ opened the pull request:
Delete the blank paragraph in the language selection button and Edit to synchronize with the English version of README.md

# What does this PR do?

<!--
Congratulations! You've made it this far! You're not quite done yet though.

Once merged, your PR is going to appear in the release notes with the title you set, so make sure it's a great title that fully reflects the extent of your awesome contribution.

Then, please replace this with a description of the change and which issue is fixed (if applicable). Please also include relevant motivation and context. List any dependencies (if any) that are required for this change.

Once you're done, someone will review your PR shortly (see the section "Who can review?" below to tag some potential reviewers). They may suggest changes to make the code even better. If no one reviewed your PR after a week has passed, don't hesitate to post a new comment @-mentioning the same persons---sometimes notifications get lost.
-->

<!-- Remove if not applicable -->

Delete an empty paragrah under the language selection by removing white space.
And synchronize `README_ko.md` to [README.md](https://github.com/huggingface/transformers/blob/main/README.md).

- [x] Additional edit for keep consistency with main [documentation](https://huggingface.co/docs/transformers/v4.44.2/ko/index). 
(메인 문서와 일관성 유지를 위한 수정)
- [x] Grammar Check (맞춤법 검사)

* Maintain the content of the existing document as much as possible.
* Juxtapose in English of some tasks.

There could be mistranslations or awkwardness.
If you found any of them, I would appreciate it if you could correct the error to make the community shine🙇‍♂️

Thanks for your hard effort that everyone can enjoy thousands of models and learn DL more easily!

I'm sorry to find this PR late #20179
(굉장히 좋은 가이드가 있었군요! 늦게 확인해서 죄송합니다😅)

## Before submitting
- [X] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#create-a-pull-request),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?


## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

<!-- Your PR will be replied to more quickly if you can figure out the right person to tag with @

 If you know how to use git blame, that is the easiest way, otherwise, here is a rough guide of **who to tag**.
 Please tag fewer than 3 people.

Models:

- text models: @ArthurZucker
- vision models: @amyeroberts
- speech models: @sanchit-gandhi
- graph models: @clefourrier

Library:

- flax: @sanchit-gandhi
- generate: @zucchini-nlp (visual-language models) or @gante (all others)
- pipelines: @Narsil
- tensorflow: @gante and @Rocketknight1
- tokenizers: @ArthurZucker
- trainer: @muellerzr and @SunMarc

Integrations:

- deepspeed: HF Trainer/Accelerate: @muellerzr
- ray/raytune: @richardliaw, @amogkam
- Big Model Inference: @SunMarc
- quantization (bitsandbytes, autogpt): @SunMarc

Documentation: @stevhliu

HF projects:

- accelerate: [different repo](https://github.com/huggingface/accelerate)
- datasets: [different repo](https://github.com/huggingface/datasets)
- diffusers: [different repo](https://github.com/huggingface/diffusers)
- rust tokenizers: [different repo](https://github.com/huggingface/tokenizers)

Maintained examples (not research project or legacy):

- Flax: @sanchit-gandhi
- PyTorch: See Models above and tag the person corresponding to the modality of the example.
- TensorFlow: @Rocketknight1

 -->

J4BEZ committed:
Update README_ko.md

Delete the blank paragraph in the language selection button and Edit to synchronize with the English version of README.md

J4BEZ committed:
Merge branch 'huggingface:main' into doc/edit_README_ko_md

J4BEZ committed:
[i18n-KO] Update README_ko.md

* Additional edit for keep consistency with main [documentation](https://huggingface.co/docs/transformers/v4.44.2/ko/index). (메인 문서와 일관성 유지를 위한 수정)

J4BEZ committed:
Update README_ko.md

Additional update.
* Change docs link to Korean translated page if it exists.

J4BEZ committed:
Change doc link to korean translated if it exists.

Change the link of doc and delete a row 'migration' of the table Learn more[더 알아보기], since it does not exist in the main version of doc.

J4BEZ committed:
modify a link of the main README.md

from
`https://huggingface.co/docs/transformers/index#supported-frameworks`

to
`https://huggingface.co/docs/transformers/index#supported-models-and-frameworks`

since the title of 'supported table' changed.

J4BEZ committed:
[i18n-ko] edit links and sync with main `README.md`

heuristicwave commented on i18n/README_ko.md:
```diff
@@ -96,24 +137,46 @@ limitations under the License.
-... })
-{'score': 0.30970096588134766, 'start': 34, 'end': 58, 'answer': 'huggingface/transformers'}
-
+# Download an image with cute cats
```
```suggestion
# 귀여운 고양이가 있는 이미지를 다운로드하세요
```

heuristicwave commented on i18n/README_ko.md:
```diff
@@ -96,24 +137,46 @@ limitations under the License.
+>>> image_data = requests.get(url, stream=True).raw
+>>> image = Image.open(image_data)
+
+# Allocate a pipeline for object detection
```
```suggestion
# 객체 감지를 위한 파이프라인을 할당하세요
```

heuristicwave reviewed (commented):
안녕하세요 @J4BEZ 님, 소중한 변역 감사합니다!

이전에 코드의 주석도 번역하라는 가이드가 있어 변경해보았습니다!

edit_README_ko 에서 README.md 파일도 수정한 것은 Update README_ko.md 과는 분리하는게 좋아보입니다!
@junejae @jungnerd @harheem @nuatmochoi 의견 부탁드려요~

J4BEZ committed:
docs/change comment to Korean1

Change English comment to Korean

Co-authored-by: Jihun Lim <31366038+heuristicwave@users.noreply.github.com>

J4BEZ committed:
docs/change comment to Korean2

Change English comment to Korean

Co-authored-by: Jihun Lim <31366038+heuristicwave@users.noreply.github.com>

J4BEZ committed:
revise to original

to seperate `edit_README_ko_md` and `README.md`

J4BEZ commented:
@heuristicwave 섬세한 피드백 너무나 감사드립니다🙇‍♂️!!
리뷰하여 주신대로 코드에 나타난 주석을 번역해주신 제안으로 수정하였으며
한국어 README.md에만 집중될 수 있도록 영문 README.md 파일의 링크는 원본으로 되돌렸습니다😆😆

이번 한 주 고생 많으셨습니다🙇‍♂️
토요일 밤, 하루 잘 마무리 하시고 복되고 평안한 주말 되시길 바랍니다🤗

J4BEZ committed:
Merge branch 'main' into doc/edit_README_ko_md

J4BEZ committed:
Merge branch 'main' into doc/edit_README_ko_md

Rocketknight1 commented:
cc @stevhliu for vis

stevhliu reviewed (approved):
LGTM, thanks for updating! Feel free to mark as ready for review when you're done 🤗

J4BEZ commented:
@stevhliu Thank you for your kind reply😄
I'll change this PR from `Draft` to `Ready for review` soon, after adjusting some improvements!

Thank you for your hard effort and hope you have a great day😄

J4BEZ committed:
Synchronization with English documentation.

Synchronization with English documentation, and translated a line of comment from English to Korean.

J4BEZ committed:
Merge branch 'main' into doc/edit_README_ko_md

J4BEZ commented:
@stevhliu Thank you for waiting😆
I finished translation and synchronization🙌🙌

Hope you have a peaceful day and stay healthy!
